flag/flag.go

```diff
@@ -54,8 +54,9 @@ func ParseSize(s, unit string) (int, error) {
 }
 
 // ParseArgs calls flag.Parse and a *Config or error.
-func ParseArgs(args []string) (*Config, error) {
+func ParseArgs(args []string) (*Config, *bool, error) {
 	c := &Config{}
+	debug := flag.Bool("debug", false, "This flag provides a list of KVM capabilities and nothing more")
```

Comment:
Could you please add a Debug column to the Config structure to adapt it to the existing parsing?
And could you define the options like this?

```go
flag.BoolVar(&c.Debug, "debug", false, "This flag provides a list of KVM capabilities and nothing more")
```

Reply:
Done